### DISCUSSION TITLE
Windows Terminal Preview v1.14.145

### DISCUSSION THREAD

**Discussion by DHowett** [microsoft/terminal#13267] [Announcements]
This release of Windows Terminal Preview, 1.14.145, was made generally available on the 26th of May, 2022.

It contains the following fixes, which are almost exclusively for bugs that we *recalled* 1.14.143 over!

As with prior releases, you need to install `Microsoft.VCLibs.UWPDesktop.140.00` or make sure it is installed prior to installing Terminal. If you are using Terminal _unpackaged_, you will need to make sure you have the systemwide "Microsoft Visual C++ Redistributable" installed, and choose the _Win10_ version of our msixbundle. Yeah, it's strange!

## Bug Fixes

* Opening a new tab or pane will no longer un-maximize, un-snap, or otherwise move Terminal around the screen (!) (#13164)
* We will no longer crash when you try to split panes in the Settings tab (#13172)
* We have changed the display language names for the different local variations of Chinese to not include location names (#13148)

<hr /><em>This discussion was created from the release <a href='https://github.com/microsoft/terminal/releases/tag/v1.14.1451.0'>Windows Terminal Preview v1.14.145</a>.</em>

**Comment by 5HT** [comment]
Is `UTF-8` fixed yet?

**Reply by zadjii-msft** [reply]
Which part of UTF-8 support are you looking for? #8000 is our big megathread tracking all unicode issues, so I'd start there.

**Reply by 5HT** [reply]
Aha, 4 of 7 tasks

**Comment by Big-Cake-jpg** [comment]
Oh no, I saw the version as 114514...... Sorry🤔

**Comment by adamlahbib** [comment]
Are tabs rounded now 😍 ?

**Reply by zadjii-msft** [reply]
They've always been rounded on the tops, rounded on the bottom is tracked as an element in #3327

**Comment by fischerdouglas** [comment]
![image](https://user-images.githubusercontent.com/7989601/174587004-642b2d95-5d0e-4c8e-b878-10a6ef728371.png)

I know it's a silly question... But:

I use the preview version of the Microsoft Store. And I've noticed many times that there is a difference between the semantic versioning code of the release that is on GitHub, and the one that comes packaged for the store.

But this time the difference caught my attention due to the coincidence of numbers.
- GitHub Release 1.14.145
- About Version (from MS Store) 1.14.1452.0

Is there any logic in this definition?

I even imagine that this question would be more directed to the team responsible for packaging for the Microsoft Store, but if I'm not mistaken, this is built into the project itself through CI/CD.

**Reply by zadjii-msft** [reply]
@DHowett We should have a doc that explains this madness. 

tl;dr - We have two slightly different packages we publish, a "windows 11" version (which actually relies on Framework Packages for dependencies, so it's a little smaller) and a "windows 10" version (which wholesale bundles them in our package, for folks who can't rely on FWP). We can't publish two versions to the store with the exact same build number, that won't do at all. So, to differentiate the "windows 11" version of the package from the "windows 10" version, we just change the last digit of the minor version. So we call the release `1.AB.XYZ`, when the packages themselves are versioned `1.AB.XYZ1` and `1.AB.XYZ2`

**Reply by fischerdouglas** [reply]
Well... Although it sounds a little crazy, as you said, it was the way they found to solve the problem, right?
For me this is acceptable!

About the documentation explaining this... I see that it is indeed necessary!

But maybe some adaptations or improvements can make this a little more self-explanatory, thus avoiding great efforts in the explanations.

I don't know exactly where the "1" and "2" came from within this pattern, but I couldn't make any direct reference to the operating system versions (after all, Windows 11 is actually its version of the operating system 10.0.xxx).
But it came to my mind that if after "1.AB.XYZ" was added a .(dot) and 10 for Windows 10, and 11 for Windows 11 this would be almost explicit.

Well, it's just a crazy idea! Take it only if it makes any sense to you!

Thank you for your patience in answering silly questions.

**Reply by zadjii-msft** [reply]
> I don't know exactly where the "1" and "2" came from within this pattern

The `1` and `2` are actually totally arbitrary.

> if after "1.AB.XYZ" was added a .(dot) and 10 for Windows 10, and 11 for Windows 11 this would be almost explicit

Man, that'd be great. We can't actually set the revision field of the package to anything other than `0` for whatever silly reason, otherwise we wouldn't have this problem at all 😅

**Comment by dmiller423** [comment]
crashtastic,   gonna build master and see if I can't debug it ...

**Comment by areavailable** [comment]
In third paragraph above :: "Microsoft.VCLibs.UWPDesktop.140.00".
Mine instead is Microsoft.VCLibs.140.00.UWPDesktop, which, in a good situation, I would assume is what was intended. 

Bad situation is that I, heretofore unaware of Windows Terminal, am troubleshooting installation failure of the .....8wekyb3d8bbwe.msixbundle file.
Immediate error window containing only : "The app didn't start". 
The end.
